Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -121,8 +119,8 @@ extension Event.HumanReadableOutputRecorder {
     guard let graph else {
       return (0, 0, 0, 0, "")
     }
-    let errorIssueCount = graph.compactMap(\.value?.errorIssueCount).reduce(into: 0, +=)
-    let warningIssueCount = graph.compactMap(\.value?.warningIssueCount).reduce(into: 0, +=)
+    let errorIssueCount = graph.compactMap { $0.value?.issueCount[.error] }.reduce(into: 0, +=)
```

Comment:
I believe this can be expressed as a keypath `compactMap(\.value?.issueCount[.error])` if you like.